### DISCUSSION TITLE
Website down at least since Nov 14th

### DISCUSSION THREAD

**Discussion by KuestenKeks** [olahallengren/sql-server-maintenance-solution#842] [General] [answered]
Hi there!

last week we noticed that https://ola.hallengren.com/ is down. It would be nice to have the documentation on the site available again :)

Cheers!

**Comment by olahallengren** [comment]
I am able to access the website. Are you getting an error, or is the website not responding?

**Reply by KuestenKeks** [reply]
Well, this is weird.. The webserver is not responding, I'm getting connection timeouts. But I've now tested with different internet connections and only one is having this issue... Using my phone did not work last week, but it works now. Well, I guess this is somehow an issue on my end, sorry about the false alarm!

![grafik](https://github.com/user-attachments/assets/5552cef8-9dea-431b-a42e-9b03d4f68e0f)

**Comment by olahallengren** [comment]
And it works to access other website from both Internet connections?

**Reply by KuestenKeks** [reply]
Yes, at least I haven't noticed any other websites that won't work so far. But I've now tested technomax.com and it has the same issue as your website.

**Reply by olahallengren** [reply]
> Yes, at least I haven't noticed any other websites that won't work so far. But I've now tested technomax.com and it has the same issue as your website.

I will check with the hosting provider. Thank you for reporting this.

**Reply by olahallengren** [reply]
Could you send me an email with your IP address (when it is not working?

**Comment by olahallengren** [comment, marked as answer]
Your IP address had got blocked, but it should be working now. Could you test again?

**Reply by KuestenKeks** [reply]
Yes, we can access your website again, thank you! Very weird.. I'd really like to know, how our IP got blocked. Was it included in some kind of blacklist?

**Reply by olahallengren** [reply]
As I understand it, there are certain things that can trigger that an IP address is getting blocked in the firewall. I can't say exactly what happened in this case. Please let me know if you experience this again.

>Was it included in some kind of blacklist?

According to the hosting provider, it was not.
